dataprofiler/profilers/profile_builder.py

```diff
@@ -840,17 +857,25 @@ def _restore_data_labelers(self, data_labeler=None):
                         dirpath=data_labeler_options.data_labeler_dirpath,
                         load_options=None)
                 self.options.set(
-                    {'data_labeler.data_labeler_object': data_labeler})
+                    {'unstructured_options.data_labeler.data_labeler_object':
+                         data_labeler})
 
             except Exception as e:
                 utils.warn_on_profile('data_labeler', e)
-                self.options.set({'data_labeler.is_enabled': False})
+                self.options.set({'unstructured_options.data_labeler.is_enabled':
+                                      False})
```

Comment:
do we need to set the structured to be disabled too?

Reply:
We should, yes